api-2.2/src/main/java/org/openmrs/module/fhir2/api/dao/impl/FhirConditionDaoImpl_2_2.java

```diff
@@ -151,7 +151,7 @@ protected void setupSearchParams(Criteria criteria, SearchParameterMap theParams
 				case FhirConstants.CODED_SEARCH_HANDLER:
 					entry.getValue().forEach(param -> handleCode(criteria, (TokenAndListParam) param.getParam()));
 					break;
-				case FhirConstants.CONDITION_CLINICAL_STATUS_HANDLER:
+				case FhirConstants.STATUS_SEARCH_HANDLER:
```

Comment:
These changes aren't part of Task resource. You should move to a different commit, PR